app/views/organizations/_reset_and_remove_organization_modal.html.erb

```diff
@@ -1,24 +1,23 @@
 <div class="remodal text-left" data-remodal-id="reset-and-remove-organization">
   <button data-remodal-action="close" class="remodal-close"><%= octicon 'x' %></button>
-  <h2 class="remodal-header">Are you ABSOLUTELY sure?</h2>
+  <h2 class="remodal-header"><%= t('views.organizations.are_you_sure') %></h2>
 
   <div class="remodal-warning">
-    Unexpected things will happen if you don't read this!
+    <%= t('views.organizations.delete_warning') %>
   </div>
 
   <div class="remodal-description">
     <p>
-      This action <strong>CANNOT</strong> be undone.
-      Please note that reseting and removing this classroom will delete all repositories and teams under the <strong><%= @organization.github_organization.login %></strong> account that were created by GitHub Classroom.
+      <%= t('views.organizations.delete_concequences', organization: @organization.github_organization.login).html_safe %>
```

Comment:
Hi @CGA1123 👋 

Thanks for the pull requests. 

One tiny nit here: `concequences` is misspelled, it should be `consequences`. 

Also, don't forget to update the [`config/locales/views/organizations/en.yml`](https://github.com/education/classroom/pull/739/files#diff-7cab1873e635f29b4f267aa5665ced5bR27) 😉